File: dohyeon/Programmers/Level_0/Lv0_특별한_이차원_배열_1.py
def solution(n) :
    answer =[]
    for i in range(n) :
        sub_list = []
        for j in range(n) :
            if i == j :
                sub_list.append(1)
            else :
                sub_list.append(0)
        answer.append(sub_list)
    return answer

print(solution(3)) # [[1, 0, 0], [0, 1, 0], [0, 0, 1]]

File: seongwon/Programmers/Level_0/Lv0_특별한_이차원_배열_1.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(int n) {
    vector<vector<int>> answer(n,vector<int>(n));
    int i = 0;
    int j = 0;
    while(i<n){
        answer[i][j] = 1;
        i++;
        j++;
    }
    return answer;
}